Build failed with Debian stable / rustc 1.70

Hi,
when building this crate with `cargo build --release` on Debian Stable (rust 1.70), a bunch of the following errors occur:

```
error[E0446]: private type `DecodeFailed` in public interface
  --> /lzxd-0.2.5/src/bitstream.rs:42:5
   |
42 |     pub fn read_bit(&mut self) -> Result<u16, DecodeFailed> {
   |     ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ can't leak private type
   |
  ::: /lzxd-0.2.5/src/lib.rs:99:1
   |
99 | enum DecodeFailed {
   | ----------------- `DecodeFailed` declared as private
```

The fix seems easy enough - declare the DecodeFailed enum as public.

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -96,7 +96,7 @@ pub struct Lzxd {
 
 /// Specific cause for decompression failure.
 #[derive(Debug, Clone, Copy, PartialEq, Eq)]
-enum DecodeFailed {
+pub enum DecodeFailed {
     /// The chunk data caused a read of more items than the current block had in a single step.
     OverreadBlock,
 

